gdal/frmts/nitf/nitfdataset.cpp

```diff
@@ -2281,7 +2281,7 @@ CPLErr NITFDataset::_SetProjection(const char* _pszProjection)
     return CE_None;
 }
 
-#ifdef ESRI_BUILD
+//#ifdef ESRI_BUILD
```

Comment:
why are those // comments needed ? This is already protected by #ifdef

Reply:
Reverted